x-pack/solutions/observability/plugins/apm_data_access/server/lib/check_privileges.ts

```diff
@@ -34,9 +32,14 @@ export async function checkPrivileges({
   const { hasAllRequested } = await checkPrivilegesFn({
```

Comment:
Do you know if behind the scene the `checkPrivilegesFn` calls `GET /_security/user/_has_privileges`? It's worth checking it before moving forward with this change.

Reply:
It does. amongst other calls

<img width="574" alt="image" src="https://github.com/user-attachments/assets/a1d28f2b-4ccb-477b-840c-d589e13e3891" />

Reply:
check this up:
<img width="1021" alt="Screenshot 2025-03-13 at 08 58 26" src="https://github.com/user-attachments/assets/264c8157-9413-4632-83ba-fd343dde5a81" />

It returns true for any index name we pass to this API. 🤷🏻‍♂️